### PR TITLE
Divine sage implementing a token on celo using golang

### PR DESCRIPTION
**Pull request by Divine572**
### Thanks for submitting this tutorial for Celo Sage
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review 
- [x] My tutorial follows the Tutorial Contribution Guidelines.
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/showcase/intermediate/deploying-celo-applications-using-golang.png
Comment (Julius170) [inline src/data-tutorials/showcase/intermediate/deploying-celo-applications-using-golang.png:1]: This image is an abberation, can you remove it

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -2726,6 +2726,22 @@ const Users: User[] = [
     website: 'blog/tutorials/building-a-celo-voting-system-with-golang',
     tags: ['celosage', 'intermediate', 'smartcontract', 'solidity'], 
 
+  },
+   {
+    title: 'Deploying Celo Applications using Golang',
```
Comment (Julius170) [inline src/data-tutorials/users.tsx:2731]: Can you correct this also, your user.tsx file

Comment (Divine572) [reply src/data-tutorials/users.tsx:2731]: Alright I'll make the changes

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**